controllers/authzctrl/controller.go

```diff
@@ -26,7 +26,7 @@ import (
 const name = "authorization"
 
 func New(cli client.Client, log logr.Logger,
-	protectedResource platform.ProtectedResource, config PlatformAuthorizationConfig) *Controller {
+	protectedResource platform.ProtectedResource, config authorization.PlatformAuthorizationConfig) *Controller {
```

Comment:
is Platform* required anymore as part of the Name?

Reply:
We can make it all `pkg.Config`. But as we're looking at naming things, maybe it's an opportunity to give it a bit more meaning. Is it Watch config? More than that?

Reply:
I think in this given case isn't it a `PlatformConfig` though? These are bits and pieces coming from the platform setup for authz components to rely on:

```golang
type PlatformConfig struct {
	// Label in a format of key=value. It's used to target created AuthConfig by Authorino instance.
	Label string
	// Audiences is a list of audiences used in the AuthConfig template when performing TokenReview.
	Audiences []string
	// ProviderName is the name of the registered external authorization provider in Service Mesh.
	ProviderName string
}
```

Reply:
PlatformConfig, ProviderConfig?